catalog-info.yaml

```diff
@@ -18,6 +18,6 @@ metadata:
     backstage.io/techdocs-ref: dir:.
     lighthouse.com/website-url: https://backstage.io
 spec:
-  type: library
+  type: service
```

Comment:
So is this unintentional or...? :)

Reply:
Flipping between to many repos doing some testing, total accident, this has been reverted.

And I added a screenshot too 👍